### PR TITLE
Fix output rendering

### PR DESCRIPTION
**Pull request by DeepikaDixit**
## Change Overview

`actionSet` fails if all output artifacts are not specified

## Pull request type

Please check the type of change your PR introduces:
- [ ] Work in Progress
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Trivial/Minor
- [X] Bugfix
- [ ] Feature
- [ ] Documentation

## Issues

- #XXX

## Test Plan

- [ ] Manual
- [ ] Unit test
- [ ] E2E

### CODE REVIEW

**Comment by DeepikaDixit** [comment]
New error when output set to `nil`:
```
time="2019-10-11T05:28:38Z" level=error msg="Failed to render output artifacts template: config:1: unexpected \"}\" in operand\nvendor/github.com/kanisterio/kanister/pkg/param.renderStringArg\n\t/repo/go/src/vendor/github.com/kanisterio/kanister/pkg/param/render.go:111\nvendor/github.com/kanisterio/kanister/pkg/param.RenderArtifacts\n\t/repo/go/src/vendor/github.com/kanisterio/kanister/pkg/param/render.go:94\nvendor/github.com/kanisterio/kanister/pkg/controller.(*Controller).runAction.func1\n\t/repo/go/src/vendor/github.com/kanisterio/kanister/pkg/controller/controller.go:447\nvendor/gopkg.in/tomb%2ev2.(*Tomb).run\n\t/repo/go/src/vendor/gopkg.in/tomb.v2/tomb.go:163\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1337"

**Comment by DeepikaDixit** [comment]
> New error when output set to `nil`:
> 
> ```
> time="2019-10-11T05:28:38Z" level=error msg="Failed to render output artifacts template: config:1: unexpected \"}\" in operand\nvendor/github.com/kanisterio/kanister/pkg/param.renderStringArg\n\t/repo/go/src/vendor/github.com/kanisterio/kanister/pkg/param/render.go:111\nvendor/github.com/kanisterio/kanister/pkg/param.RenderArtifacts\n\t/repo/go/src/vendor/github.com/kanisterio/kanister/pkg/param/render.go:94\nvendor/github.com/kanisterio/kanister/pkg/controller.(*Controller).runAction.func1\n\t/repo/go/src/vendor/github.com/kanisterio/kanister/pkg/controller/controller.go:447\nvendor/gopkg.in/tomb%2ev2.(*Tomb).run\n\t/repo/go/src/vendor/gopkg.in/tomb.v2/tomb.go:163\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1337"
> ```

This is blueprint error 🤦‍♀

File: pkg/function/describe_backups.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func describeBackupsPodFunc(cli kubernetes.Interface, tp param.TemplateParams, n
 					DescribeBackupsFileCount:         nil,
 					DescribeBackupsSize:              nil,
 					DescribeBackupsPasswordIncorrect: nil,
-					DescribeBackupsRepoDoesNotExist:  "false",
+					DescribeBackupsRepoDoesNotExist:  "true",
```
Comment (DeepikaDixit) [inline pkg/function/describe_backups.go]: This was my bad. It needs to return true, previous PR made it false.

File: pkg/function/describe_backups.go [hunk 2]
```diff
@@ -106,7 +106,7 @@ func describeBackupsPodFunc(cli kubernetes.Interface, tp param.TemplateParams, n
 					DescribeBackupsFileCount:         nil,
 					DescribeBackupsSize:              nil,
 					DescribeBackupsPasswordIncorrect: nil,
```
Comment (vkamra) [inline pkg/function/describe_backups.go]: Should we set this to false also?

Comment (DeepikaDixit) [reply pkg/function/describe_backups.go]: Sure. That might be better.

**Pull request merged**